db/migrate/20210525143303_add_docx_preview_processing_to_reports.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDocxPreviewProcessingToReports < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
You can make previews just after docx generation thus no additional flag would be needed